docs/_layouts/default.html

```diff
@@ -62,7 +62,7 @@ <h1><a href="{{ "/" | absolute_url }}">
     </div>
     <script src="{{ "/assets/js/scale.fix.js" | relative_url }}"></script>
     <script type="text/javascript">
-        var articleUrlPrefix = "{{ site.url }}/kb/";
+        var articleUrlTemplate = "{% link kb/KHV002.md %}";
```

Comment:
`link` does not include site base url.
You should prepend it to the link:
`var articleUrlTemplate = "{{ site.baseurl }}{% link kb/KHV002.md %}";`

Reply:
This was a confusing one. When I tested this locally it worked fine. Also when I read the documentation on the `link` tag, it said that adding the base url is not required since Jekyll `4.0.0`: https://github.com/jekyll/jekyll/blame/master/docs/_docs/liquid/tags.md#L88 but then I run a test on another GitHub Pages and saw that the generated link was missing the base url.
[Turns out](https://pages.github.com/versions/) that the Jekyll version that GitHub uses now is `3.8.5` which explains why this is required. 
Thanks for catching this!